HelpSource/Classes/PV_ChainUGen.schelp

```diff
@@ -1,52 +1,135 @@
 class:: PV_ChainUGen
-summary:: Base class for UGens that alter FFT chains
+summary:: Base class for UGens that operate on FFT chains
 categories:: UGens>FFT
 
 description::
-"PV_ChainUGen" is an abstract class - in other words, a class that you do not use directly. Instead, use one of its subclasses. It represents phase-vocoder UGens - i.e. UGens which apply some kind of transformation to the frequency-domain signal produced by FFT.
+code::PV_ChainUGen:: is an abstract class, not used directly, but only its subclasses. It represents phase-vocoder UGens - i.e. UGens which apply some kind of transformation to the frequency-domain signal produced by link::Classes/FFT::.
```

Comment:
Maybe "is an abstract class. code::PV_ChainUGen:: is not used directly; only its subclasses are." As it's written now it's not quite clear what the last part of this sentence means

Reply:
better!